.github/workflows/register_repositories.py

```diff
@@ -15,8 +15,33 @@
 XDEVBOT_MAIN_ENDPOINT = 'http://xdevbot.herokuapp.com/'
 
 
+def validate_repo_campaign_info(info, config):
+    error_messages = []
+    valid = True
+    expected_keys = {'campaign', 'repo'}
+    if set(info.keys()) == expected_keys:
+        if info['campaign'] not in config:
+            error_messages.append(
+                f"\n  - Unknown campaign `{info['campaign']}`. Valid campaigns include `{', '.join(config.keys())}`."
+            )
+            valid = False
+
+        if not info['repo']:
+            valid = False
+            error_messages.append('\n  - No specified repository')
+
+    else:
+        valid = False
+        error_messages.append(
+            f'\n  - Found unexpected keys `({", ".join(info.keys())})` in the parsed command. Expected keys are {", ".join(expected_keys)}.'
```

Comment:
Can we back tick the expected keys, too?

Reply:
Done